migrations/20241103183508-migrate-expense-policies-to-policies.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+      UPDATE "Collectives" SET
+      data = jsonb_set(
+        data,
+        '{policies,EXPENSE_POLICIES}',
```

Comment:
We need a `COALESCE` here, as `JSONB_SET(null, ...)` gives `NULL`. We'd be dropping existing policies.

```sql
SELECT COUNT(*)
FROM "Collectives"
WHERE TRIM("expensePolicy") <> '' AND "expensePolicy" IS NOT NULL
AND data IS NULL
```

=> 126 entries